### PR TITLE
The next web responsive replica

### PR DESCRIPTION
**Pull request by jaimdedan**
- Create a navbar with dropdown
- Use grid for cover section
- Use flexbox for other sections
- Make it responsive for width of 360px, 768px, 972px

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1111 @@
...
+                </li>
+              </ul>
+            </div>
+            <div class="most-popular-card">
+              <h2 class="heading">Hard Fork ></h2>
+              <article class="cover cover-1">
+                <div class="gradient"></div>
+                <div class="gradient2"></div>
+                <div class="background">
+                  <!-- <img src="imgs/7.png" alt=""> -->
```
Comment (krys2fa) [inline index.html:444]: - [ ] [REQUIRED] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
Same applies for lines **511**, **578**, **645**, **712**, **779**, **846** and **913**

**Review by krys2fa — CHANGES_REQUESTED** [review]
## You've done really well team. 👏 
Your project displays good use of semantic HTML as well as CSS classes. 💪 
You have a professional-looking README file as well. 👍 

### Status: Changes Required ♻️ 

- [ ] [REQUIRED] All elements should be aligned with [float](https://css-tricks.com/all-about-floats/) and [flex](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) or [grid](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/responsive_design/align_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540217/)).
You used both flex and grid but not float. You were expected to use float and either one of flex or grid. Please employ float in aligning one or more elements.

- [ ] [OPTIONAL] The social media icons in your page look different than in the original. The difference is shown below. Please fix this so the clone looks similar to the original

Original Page          |  Your Page
:-------------------------:|:-------------------------:
Screen Shot 2020-12-02 at 5 18 08 PM |  Screen Shot 2020-12-02 at 5 18 18 PM

Please request for another review after implementing the changes.

Happy Coding 🖥️  😃 

###### You can contact me on slack (Chris Amanor) for any clarification.

**Comment by jaimdedan** [comment]
- Add float to the nav-right element
- Change the color of the footer social links background
- Edit the README.md acknowledgment section
- Delete the commented code in HTML

**Review by Spayco — APPROVED** [review]
# Project Approved

thank you for making the required changes, your project is now approved!

### please don't forget to merge to master, good luck with the next projects!


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Pull request merged**